ScoreSaber/UI/Leaderboard/ScoreSaberLeaderboard.cs

```diff
@@ -0,0 +1,44 @@
+﻿using HMUI;
+using LeaderboardCore.Managers;
+using LeaderboardCore.Models;
+using ScoreSaber.Core.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ScoreSaber.UI.Leaderboard {
+    internal class ScoreSaberLeaderboard : CustomLeaderboard, IDisposable {
+
+        private readonly CustomLeaderboardManager _manager;
+        private readonly ScoreSaberLeaderboardViewController _leaderboardView;
+        private readonly PlayerService _playerService;
+
+        public override bool ShowForLevel(BeatmapKey? selectedLevel) {
+            if (selectedLevel.HasValue) {
+                if (!selectedLevel.Value.levelId.Contains("custom_level_")) {
+                    return false;
+                }
+            }
+            return true;
```

Comment:
same thing about customs detection and possible inlusion of OST/DLC here.

Reply:
Will have to wait on umbra to say something on it